pkg/database/memory/UserModel.go

```diff
@@ -0,0 +1,51 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/quan-to/chevron/pkg/models"
+	"strings"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).